src/System.Management.Automation/engine/hostifaces/MshHostUserInterface.cs

```diff
@@ -620,13 +620,10 @@ internal void TranscribeResult(Runspace sourceRunspace, string resultText)
 
                     resultText = resultText.TrimEnd();
 
-                    if (ExperimentalFeature.IsEnabled("PSAnsiRendering"))
+                    var text = new ValueStringDecorated(resultText);
```

Comment:
Side note: the struct could be readonly struct.

Reply:
making the struct readonly requires making the fields readonly which breaks the deferred creation of PlainText only when needed

Reply:
LazyInitializer could be used for PlainText.